installing/installing_openstack/installing-openstack-user.adoc

```diff
@@ -38,8 +38,10 @@ include::modules/installation-osp-describing-cloud-parameters.adoc[leveloffset=+
 include::modules/installation-user-infra-generate.adoc[leveloffset=+1]
 include::modules/installation-initializing.adoc[leveloffset=+1]
 include::modules/installation-configuration-parameters.adoc[leveloffset=+1]
-include::modules/installation-osp-custom-subnet.adoc[leveloffset=+2]
 include::modules/installation-osp-config-yaml.adoc[leveloffset=+2]
+include::modules/installation-osp-custom-subnet.adoc[leveloffset=+2]
+include::modules/installation-osp-deploying-bare-metal-preexisting.adoc[leveloffset=+2]
+include::modules/installation-osp-deploying-all-bare-metal-preexisting.adoc[leveloffset=+2]
```

Comment:
These two modules aren't specific to UPI, they're also relevant to IPI with custom subnet.